### DISCUSSION TITLE
KEDA with azure-logs-analytics: sometimes lost metrics

### DISCUSSION THREAD

**Discussion by barside-code** [kedacore/keda#2996] [Q&A / Need Help]
Hi everyone,
in a production environment we're observing this issue: sometimes the KEDA metric server is unable to find metric calling azure-log-analytics (I suppose for timeout in Azure API invocation) and by default KEDA use a CPU metric.

This is one tipic error:

Warning  FailedGetExternalMetric       30s (x92 over 17h)  horizontal-pod-autoscaler  unable to get external metric <NAMESPACE>/azure-log-analytics-XXXXXXXXXXXX/&LabelSelector{MatchLabels:map[string]string{scaledobject.keda.sh/name: procedura-consumer-keda,},MatchExpressions:[]LabelSelectorRequirement{},}: unable to
fetch metrics from external metrics API: No matching metrics found for azure-log-analytics-XXXXX

After a while, the metric came back.

Can you please healp us to find a fine tuning for optimize this scenario?
Regards,

**Comment by zroubalik** [comment]
@tomkerkhove CC

**Comment by tomkerkhove** [comment]
Can you share your scaler configuration please? How long is the metric "gone"?
